chromium/storage.js

```diff
@@ -1,7 +1,7 @@
-var storage = chrome.storage.local;
+const storage = chrome.storage.local;
 if (chrome.storage.sync) {
   storage = chrome.storage.sync;
```

Comment:
`TypeError: invalid assignment to const 'storage'`

I also see other errors during testing. Please test and submit fixes.